lib/bootloader_setup.pm

```diff
@@ -35,17 +36,32 @@ our @EXPORT = qw(
   tianocore_select_bootloader
 );
 
+# prevent grub2 timeout; 'esc' would be cleaner, but grub2-efi falls to the menu then
+sub stop_grub_timeout {
+    send_key 'up';
+}
+
 sub boot_local_disk {
     if (check_var('ARCH', 'ppc64le')) {
         # TODO use bootindex to properly boot from disk when first in boot order is cd-rom
         wait_screen_change { send_key 'ret' };
-        if (check_screen 'inst-slof', 5) {
-            # specify local disk for boot
+        assert_screen [qw(inst-slof bootloader grub2 inst-bootmenu)];
```

Comment:
;-) It actually also solves a problem which I observed because of the timeout value of 5 seconds. Of course, we could have just blamed nested virt testing ppc64le worker :-)
And I agree that the partial repetition of tags is not that nice and code might change in near future after this PR.